src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -5489,7 +5490,63 @@ public void testListNodes() throws Exception {
       assertThat(plugin.name(), allOf(notNullValue(), not("")));
     }
   }
-  
+
+  @Test
+  public void testInspectNode() throws Exception {
```

Comment:
+1 to what david said - with the unit test, you can test that each field has a certain expected value, whereas with this test you can only really test really broad things about the response.